en_US/messages.json

```
@@ -78,5 +78,35 @@
     "qualityEstimationMessage": {
       "message": "Enable quality estimation",
       "description":"Checkbox to enable the quality estimation of the translation."
+    },
+    "formtranslationsReady": {
+      "message": "Ready",
+      "description":"Notifies that the form translation is ready to be used."
+    },
+    "formtranslationsTyping": {
+      "message": "Typing",
```

Looking at the screencast, I think this should have ellipsis, because it shows an action in progress.
```suggestion
      "message": "Typing…",
```